examples/petsc/ex5p.cpp

```diff
@@ -210,7 +210,7 @@ int main(int argc, char *argv[])
    PetscParMatrix *pM = NULL, *pB = NULL, *pBT = NULL;
    HypreParMatrix *M = NULL, *B = NULL, *BT = NULL;
    Operator::Type tid =
-      !use_petsc ? Operator::HYPRE_PARCSR :
+      !use_petsc ? Operator::MFEM_HYPRE_PARCSR :
```

Comment:
The `MFEM_` prefix is redundant  - `Operator` is already in the `mfem` namespace - so I prefer to use something like `Hypre_ParCSR` where not all characters are capitalized.

Reply:
I'm OK with that, as long as _hypre_ doesn't decide to define a public macro for this too. I agree that it is sub-optimal, but that's the reason for the `MFEM_` prefix.

Reply:
I agree with Tzanio - that the change is fine, so long as hypre doesn't change as well.  I prefer the MFEM_ prefix myself...